Don't replace agent logs in migration (#465)
ostis-ai/sc-machine: docs/sc-tools/migrate_to_new_agent_api.md

Example of usage:
```shell
./scripts/migration/migrate_to_new_agent_api.sh path/to/folder/with/source/files
```

!!! note

>     Pass path to source files that does not contain other projects as subfolders.

Reviewer: ```suggestion
    Provide path to source files that does not contain subfolders with other projects.
```

>     If path contains other projects as subfolders then those projects may be reformatted as well.

Reviewer: I thinks it may be unclear for users.

I propose this variant:
`If there are subfolders with other projects in the folder at the specified path, sources of these projects will be modified as well.`

Author: tried to update note

Reviewer: I wouldn't use the `reformat` term because reformatting refers to changing the layout of a text without any effect on its content. Maybe it had better to use the `change` (`fix`) term.
